tutorials/nwbc-cflp-integration/nwbc-cflp-integration.md

```diff
@@ -0,0 +1,131 @@
+SAP ONE Support Launchpad ([SAP ONE Support Launchpad](https://launchpad.support.sap.com/#/softwarecenter/template/products/%20_APP=00200682500000001943&_EVENT=DISPHIER&HEADER=Y&FUNCTIONBAR=N&EVENT=TREE&NE=NAVIGATE&ENR=73554900100200012051&V=MAINT&TA=ACTUAL&PAGE=SEARCH/SAP%20BUSINESS%20CLIENT%207.70))
+- You have created a **trial account on SAP BTP**:  [Get a Free Account on SAP BTP Trial](hcp-create-trial-account)
+- You are subscribed to the SAP Build Work Zone, standard edition including the Launchpad Admin role: see the following tutorial [Deliver Your First SAP Fiori Launchpad Site] (mission.cp-starter-digitalexp-portal)
+
+See also: [SAP Build Work Zone, standard edition (formerly known as SAP Launchpad Service) on BTP](https://help.sap.com/docs/Launchpad_Service/8c8e1958338140699bd4811b37b82ece/fd79b232967545569d1ae4d8f691016b.html?locale=en-US)
+
+## Intro
+
+**Additional Information: Direct Access and Tunneled Access**
+
+Direct access is only possible within the corporate network (or via VPN).
+
+Tunneled Access is possible from the corporate network or from the Internet (for which you need the Cloud Connector). The Cloud Connector is http/https based. The DIAG protocol of SAP GUI for Windows cannot be used. We are therefore using direct access here.
+
+**Procedure**
+The following steps will show you how to launch SAP GUI for Windows applications directly from the FLP@BTP. As an example, we like to create an app/tile in the
+SAP Build Work Zone, standard edition (FLP@BTP) to open an SAP GUI for Windows transaction from a remote system (here U1Y*). To connect, you must create a http destination to this system (step 1 below).
+
+>In general, there are two ways of integrating content into the
+SAP Build Work Zone, standard edition: Manual Integration of Business Content, and integration of content from remote content providers. In this blog, we demonstrate the manual integration of apps. Administrators who need to integrate hundreds of roles will use the federation approach, described at the end of this blog.
+
+
+### Create runtime destinations to access apps
+To launch apps in the
+SAP Build Work Zone, standard edition, the service must know where the content/apps are located. It is therefore necessary to create a destination to tell the BTP where your app is running.
+
+1. Logon to the BTP cockpit and open the Destination editor (subaccount=>Overview=> Connectivity=>Destinations)
+
+
+2. Choose **New Destination** and enter the name, type, description (optional), and URL as shown in the screenshot below. You can copy the URL from the landscape file or directly from Business Client (system selector, right click).
+
+```
+ https://<server>.<domain>.<ext>:<port>.
+
+```
+
+
+   ![Destination configuration](1.png)
+
+ 3. Configure destinations and add additional properties as shown in the screenshot below. For example sap-login-group, message server and instance number, sap-snc-name, can be found in SAP Logon.
```

Comment:
[...]below, for example, sap-login-group,[...]
put sap-snc-name in back-ticks

Reply:
In the markdown, it would look like this: `sap-snc-name`